File: C C++/Ago-31-2021/ex003.c
#include<stdio.h>
int main(){
    float n1,n2;
    printf("Distância percorrida (Km): ");
    scanf("%f", &n1);
    printf("Combustivel (L): ");
    scanf("%f", &n2);

    printf("O consumo médio é: %.2f Km/L", n1/n2);
    return 0;
}

File: _java/ex003.java
// Kilometragem por litro
package Uniso._java;

import java.util.Scanner;

public class ex003 {
    public static void main(String[] args) {
        Scanner scanf = new Scanner(System.in);
        float n1, n2;

        System.out.print("Distância percorrida (Km): ");
        n1 = scanf.nextFloat();
        System.out.print("Litros de combustível: ");
        n2 = scanf.nextFloat();

        System.out.print("O consumo médio é: "+ (n1/n2) +" Km/L");
        scanf.close();
    }
}
